9.1/paper.tex

```diff
@@ -242,14 +242,34 @@ \section{Overview}
   N{\'e}d{\'e}lec element on quadrilaterals and hexahedra. It is based on
   the work of Zaglmayr \cite{Zag06} and overcomes the sign conflict issues
   present in traditional N{\'e}d{\'e}lec elements that arise from the edge
-  and face parameterizations used in the basis functions. Therefore, this
+  and face parameterizations used in the basis functions.
+  Because of the sign conflict it is only possible to use the traditional
+  N{\'e}d{\'e}lec element either in a grid configuration or in a complex
+  tetrahedra mesh configuration. It is not possible to use traditional
+  N{\'e}d{\'e}lec in a complex hexahedra mesh configuration. This
```

Comment:
I don't think that's generically true. It's true that the *implementation in deal.II* has difficulties with this, though that is not the fault of "the Nedelec element".